website/src/_posts/2018-10-30-beam-summit-aftermath.md

```diff
@@ -0,0 +1,111 @@
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+<br/>
+#### The portability layer was a big topic:
+<blockquote class="twitter-tweet" data-lang="nl"><p lang="en" dir="ltr">Excellent talk by <a href="https://twitter.com/stadtlegende?ref_src=twsrc%5Etfw">@stadtlegende</a> on adding portability to <a href="https://twitter.com/hashtag/ApacheBeam?src=hash&amp;ref_src=twsrc%5Etfw">#ApacheBeam</a>, awesome milestone and next step to make the Apache Beam vision become a reality! <a href="https://t.co/M9jERlTeAE">pic.twitter.com/M9jERlTeAE</a></p>&mdash; Matthias Feys (@FsMatt) <a href="https://twitter.com/FsMatt/status/1047105336841244673?ref_src=twsrc%5Etfw">2 oktober 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+<br/>
+#### As well as a session on how to build your own SDK:
+<blockquote class="twitter-tweet" data-lang="nl"><p lang="en" dir="ltr">Robert Bredshaw explains how to build a new <a href="https://twitter.com/ApacheBeam?ref_src=twsrc%5Etfw">@ApacheBeam</a> SDK.<a href="https://twitter.com/hashtag/BeamSummit?src=hash&amp;ref_src=twsrc%5Etfw">#BeamSummit</a> <a href="https://t.co/Bj84GJimdo">pic.twitter.com/Bj84GJimdo</a></p>&mdash; Maximilian Michels 🧗 (@stadtlegende) <a href="https://twitter.com/stadtlegende/status/1047139320195366912?ref_src=twsrc%5Etfw">2 oktober 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+## Presentations
+In the aftermath of the Summit, you can check the presentations of all the sessions.
+
+### Day 1: Use cases
+* [Day 1 - Session 1 - Large scale stream analytics with Apache Beam at Sky](https://drive.google.com/open?id=1hyHw7RVpFrFpli3vLt6JGBHrEm4BcgF-5nRdH1ZE8qo)
+* [Day 1 - Session 2 - Running Quantitative Analytics with Apache Beam](https://drive.google.com/open?id=1MxYrFDVoVFsrzbTtmr18zcbPFUU4nSdi)
+* [Day 1 - Session 3 - Talend Data Streams: Building Big Data pipelines with Apache Beam](https://drive.google.com/open?id=0B4bFLXEWuluSdVBJSnZrbTZjSGFHbnd4cExYOGZQU2hmY3lF)
+* [Day 1 - Session 4 - Lesson Learned from Migrating to Apache Beam for Geo-Data Visualisation](https://drive.google.com/open?id=1-GIUVn9QBtg6t-O8uINDkMO4PyZSU_HAEjMWuUHiYY4)
+
+### Day 2: Beginners track
+* [Day 2 - Beginner - Session 1 - Development Environment with Apache Beam](https://drive.google.com/open?id=1aFH6lhnVIq4Alu-_HItQ0QOddEPJQRqI5jV_t0o3CYI)
```

Comment:
@matthiasa4 just noticed that links in this section all point to the same document. Can you please fix this?

Reply:
Oops! Thanks for spotting @tweise. Please find the right links in this PR: https://github.com/apache/beam/pull/6935